lib/request.js

```diff
@@ -1,12 +1,24 @@
 'use strict'
 
+// Copied from https://github.com/delvedor/find-my-way/blob/master/index.js#L378
+function sanitizeUrl (url) {
+  for (var i = 0, len = url.length; i < len; i++) {
+    var charCode = url.charCodeAt(i)
+    if (charCode === 63 || charCode === 35) {
+      return url.slice(0, i)
+    }
+  }
+  return url
+}
```

Comment:
can we get this out from find-my-way?

Reply:
Unfortunately `find-my-way` don't set it into `req`.
See [here](https://github.com/delvedor/find-my-way/blob/master/index.js#L187)